scripts/run_mypy_checks.py

```diff
@@ -18,16 +18,18 @@
 
 from __future__ import annotations
 
-
 import argparse
 import os
 import site
 import subprocess
 import sys
 
+
```

Comment:
Why is this line added?

Reply:
Maybe previously added by mistake. removed this time.